src/internal/messaging/solana-messaging.ts

```diff
@@ -247,6 +255,14 @@ export class SolanaThread implements Thread {
   get updatedAt() {
     return new Date(this.dialectAccount.dialect.lastMessageTimestamp);
   }
+
+  setLastReadMessageTime(time: Date): Promise<void> {
+    return Promise.resolve(undefined);
+  }
+
+  get hasUnreadMessages(): boolean {
+    return false;
```

Comment:
TODO?

Reply:
Actually this is unused, removing for now